puppet/zulip/files/nagios_plugins/zulip_postgresql/check_postgresql_replication_lag

```diff
@@ -33,7 +37,10 @@ def run_sql_query(query: str) -> List[List[str]]:
         "-z",  # Separate columns with nulls
         "-v",
         "ON_ERROR_STOP=1",
-        "zulip",
+        "-d",
+        get_config(config_file, "postgresql", "database_name", "zulip"),
+        "-u",
+        get_config(config_file, "postgresql", "datbase_user", "zulip"),
```

Comment:
This mispells `database_user`; I guess this slipped through since we don't test this commit.

I think it'd be a good idea to try splitting this into several smaller commits.  I would happily merge a PR with 5 commits that each adjust a single script like this one to respect `database_name` and `database_user`, without hooking it all together in the installer; that will let us shrink the complexity of this PR to just be about the few larger issues.

Reply:
Updated the typo, but not quite sure how to split this up to ensure each commit is self-sufficient and won't break if merged on its own.